src/convertToObject.js

```diff
@@ -12,7 +12,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+
+  const trimmedArray = sourceArray.map(word => (word + '').trim());
+
+  const arrayWithValues = trimmedArray.map(string => string.split(':'));
+
+  const arrayWithoutEmptyValues = arrayWithValues.filter(value =>
+    value.length > 1);
+
+  const property = {};
+
+  function callback(array) {
+    property[array[0].trim()] = array[1].trim();
+
+    return property;
+  }
+
+  const result = arrayWithoutEmptyValues.map(callback)[0];
```

Comment:
Зараз у тебе буде масив з одинаковими об'єктами, maybe reduce ?
Або пройдися циклом по масиву і постворюй нові властивості в об'єкті

Reply:
незрозуміло, який саме результат
```suggestion
  const styles = arrayWithoutEmptyValues.map(callback)[0];
```

Reply:
done